sdk/tests/utilities/credentials_source.py

```diff
@@ -60,8 +64,8 @@ def fetch_credentials(cls):
                 if value is None:
                     vars[key] = config_vars[key]
 
-        if None in vars.values():
-            assert False, "Source test configuration missing values from both secrets file and environment variables"
+        # if access_token is not None and None in vars.values():
```

Comment:
why was this removed, looks like it is still needed

Reply:
Agreed. Updated.